extensions/amp-story-interactive/0.1/amp-story-interactive-img-quiz.css

```diff
@@ -13,3 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-img-quiz-answer-choice-container {
+  position: absolute !important;
+  height: 1.5em !important;
+  width: 1.5em !important;
+  bottom: 0.5em !important;
+  right: 0.5em !important;
+  border: 0.125em solid var(--i-amphtml-interactive-option-background-color) !important;
+  background-color: var(--interactive-prompt-background) !important;
+  border-radius: 50% !important;
+  display: flex !important;
+  justify-content: center !important;
+  align-items: center !important;
+  text-align: center !important;
```

Comment:
You can remove all these properties, and on the ABCD add
```css
... {
  text-align: center !important;
  line-height: 1.5em !important;
}
```

then you can remove the `position: absolute; left: ...; right: ...` on the ABDC as well. You'll need a `position: absolute` on the `::before`

This will avoid using flexbox layouts without the need for them, and simplify the code